blobfuse2-release.yaml

```
@@ -392,26 +497,121 @@ stages:
             inputs:
               artifactName: 'blobfuse2-signed'
               downloadPath: $(root_dir)
-              itemPattern: blobfuse2-signed/blobfuse2*$(tags)*.deb
+              itemPattern: blobfuse2-signed/blobfuse2*$(tags).xdeb
 
           - script: |
-              for f in ./blobfuse2*$(tags)*.deb; do mv -v "$f" "${f/-$(tags)./-$(vmImage).}"; done;
-              cp ./blobfuse2*$(vmImage)*.deb $(Build.ArtifactStagingDirectory)
+              for f in ./blobfuse2*$(tags)x86_64.deb; do mv -v "$f" "${f/-$(tags)./-$(vmImage).}"; done;
+              cp ./blobfuse2*$(vmImage)*x86_64.deb $(Build.ArtifactStagingDirectory)
```

Remove this and the changes below relate to `x86_64` as the rename is already done in the above line.
```suggestion
              cp ./blobfuse2*$(vmImage)*.deb $(Build.ArtifactStagingDirectory)
```